exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,9 @@
-// Write your function here
+function text(greeting, name) {
```

Comment:
Looking at the line provided: `var greeting = createGreeting("Daniel");`

We want to write a function called `createGreeting` that accepts one parameter (the person's name) then returns the full greeting. Can you rewrite this function to do that?